api/browser.ts

```diff
@@ -25,9 +25,7 @@ export class KastleBrowserAPI {
 
   constructor() {}
 
-  async connect(
-    networkId: "mainnet" | "testnet-10" = "mainnet",
-  ): Promise<boolean> {
+  async connect(): Promise<boolean> {
```

Comment:
So a dApp dev will have to first switch network and then connect? If the user is on mainnet and dApp is requesting connection to testnet.

Reply:
Correct, the first thing is to confirm the current wallet network is as same as the dApp target network for the dApp